frontend/GoodGrocer/app/screens/Login.js

```diff
@@ -29,32 +24,33 @@ const Login = ({ navigation }) => {
       password,
     },
     onSuccess: (data) => {
-      if (data.token) {
-        dispatch(setToken(data.token));
-      } else {
-        setFailedLogIn(true);
-      }
-      console.log(data);
-    },
-    onFail: () => {
-      setFailedLogIn(true);
+      dispatch(setToken(data.token));
     },
   });
 
   return (
     <SafeAreaView style={styles.container}>
       <View style={{ paddingTop: 50 }}>
-        {/* <Image source={require("../assets/logo.png")} /> */}
         <Text style={styles.titleText}>Welcome Back</Text>
         <Text>Email or Phone Number</Text>
```

Comment:
```suggestion
        <Text>Email</Text>
```
We don't allow sign in with phone number right?

Reply:
Yup, good catch